### ISSUE TITLE
Accessibility issues in Table, Column components

### ISSUE THREAD

**Issue by nmgokhale** [bvaughn/react-virtualized#1808]
## Bug Report

Accessibility violations in Table, Column, Grid components - 

- **The tabbable element's role 'columnheader' is not a widget role**
```
<div
    style="flex: 0 1 60px;"
    tabindex="0"
    role="columnheader"
    class="ReactVirtualized__Table__headerColumn _2F2N2luOrTSa1tMUcxFhGG ReactVirtualized__Table__sortableHeaderColumn"
    aria-sort="ascending"
    aria-label="Index">
```

- **The element with role "rowgroup" is not contained in or owned by an element with one of the following roles: "grid, table, treegrid"**
```
<div
    style="width: auto; height: 40000px; max-width: 969px; max-height: 40000px; overflow: hidden; position: relative;"
    role="rowgroup"
    class="ReactVirtualized__Grid__innerScrollContainer">
```

### What is the current behavior?

Steps to reproduce -
1. Add accessibility checker extension in browser - https://www.ibm.com/able/toolkit/verify/automated/
2. Go to page - https://bvaughn.github.io/react-virtualized/#/components/Table
3. Open browser's developer tools. Go to tab - `"Accessibility Assessment"`
4. Click on `"Scan"` button
5. After scan completes, filter the results to show only `"Violations"`.

![react-virtualized-table-violations](https://user-images.githubusercontent.com/25124000/224129520-7e7d1150-d025-4617-9a15-f51d991f74d1.png)

### What is the expected behavior?
- **The tabbable element's role 'columnheader' is not a widget role** : Allow an option to set tabIndex="-1" in [Column component](https://github.com/bvaughn/react-virtualized/blob/master/docs/Column.md)
- **The element with role "rowgroup" is not contained in or owned by an element with one of the following roles: "grid, table, treegrid"**: Allow an option to set role in [Table component](https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md)

### Which versions of React and react-virtualized, and which browser / OS are affected by this issue? 
Did this work in previous versions of react-virtualized? - No
|                   |          |
|-------------------|----------|
| Browser           |   Google Chrome       |
| OS                |      macOS    |
| React             |     16.13.1     |
| React DOM         |     16.13.1     |
| react-virtualized |     9.22.3    |

### Related issues - 
https://github.com/bvaughn/react-virtualized/issues/1604

CC: @bvaughn

**Comment by nmgokhale** [comment]
@bvaughn Is this issue fixed in version [9.22.4](https://github.com/bvaughn/react-virtualized/releases/tag/v9.22.4) ?
